adafruit_platformdetect/chip.py

```diff
@@ -22,6 +22,7 @@
 HFU540 = "HFU540"
 MCP2221 = "MCP2221"
 BINHO = "BINHO"
+ONION = "ONION_OMEGA"
```

Comment:
Let's change this line to something more like:
`MIPS24KC = "MIPS24KC"`

Reply:
Done! :yum: